src/space_stage/SpaceStage.tscn

```diff
@@ -5,7 +5,7 @@
 [ext_resource type="PackedScene" uid="uid://c2h4vpmx0juq8" path="res://assets/models/menu_backgrounds/StarsMesh.tscn" id="3"]
 [ext_resource type="PackedScene" uid="uid://dokew8lhrri5" path="res://assets/models/menu_backgrounds/Comet.tscn" id="4_aqo7l"]
 [ext_resource type="PackedScene" uid="uid://i8evqpkyb68h" path="res://src/space_stage/gui/SpaceHUD.tscn" id="7"]
-[ext_resource type="PackedScene" uid="uid://devtugnggmuol" path="res://src/general/PauseMenu.tscn" id="8"]
+[ext_resource type="PackedScene" uid="uid://devtugnggmuol" path="res://src/gui_common/menus/PauseMenu.tscn" id="8"]
```

Comment:
I can kind of see why this change makes logical sense, *however* my future plan is that the pause menu would be a persistent scene that is always loaded and not a part of all the separate scenes (https://github.com/Revolutionary-Games/Thrive/issues/4805). So this is again a change I would like to not have in this PR.